packages/cli/src/services/ownership.service.ts

```diff
@@ -179,4 +192,67 @@ export class OwnershipService {
 			where: { role: { slug: GLOBAL_OWNER_ROLE.slug } },
 		});
 	}
+
+	async hasInstanceOwner() {
+		return await this.userRepository.exists({
+			where: [
```

Comment:
What about:
```ts
where: {
			role: { slug: GLOBAL_OWNER_ROLE.slug },
			// OR conditions:
			// 1) lastActiveAt IS NOT NULL
			// 2) password IS NOT NULL
			// (PAY-4247 workaround)
			OR: [
				{ lastActiveAt: Not(IsNull()) },
				{ password: Not(IsNull()) },
			],
		},
```

Reply:
Ah yes, my bad you're right, i think i had the query builder syntax in mind. good for me then 👍